package.json

```diff
@@ -36,10 +39,11 @@
   "dependencies": {
     "backbone": "^1.4.0",
     "flux": "^3.1.3",
-    "jquery": "^3.4.0",
+    "jquery": "^1.8.0",
```

Comment:
this seems like a pretty big jump backwards in jquery histrory -- anything to be concerned about?

Reply:
> hrm are you sure @hong4rc ? once I ran `yarn` to upgrade, I was able to get everything building just fine. its also in the prod branch right now:
> [`gh-pages` (commits)](https://github.com/pcottle/learnGitBranching/commits/gh-pages)
> 
> and everything seems to be working great

:)
 You forgot call `yarn` to install `jquery v1.8`
![image](https://user-images.githubusercontent.com/19208123/79929777-e74ce000-8470-11ea-86ce-0f803e3903cc.png)

Reply:
For reference, there's further discussion about this in #681.